testing/trino-product-tests/README.md

```diff
@@ -34,7 +34,7 @@ broken.
 * Install [Docker for Mac](https://docs.docker.com/docker-for-mac/)
 
 * Add entries in `/etc/hosts` for all services running in docker containers:
-`hadoop-master`, `postgres`, `cassandra`, `presto-master`.
+`hadoop-master`, `postgres`, `cassandra`, `trino-master`.
```

Comment:
@wendigo do we still use `trino-master` or now `coordinator`?

Reply:
`presto-master` this still is -- https://github.com/trinodb/trino/blob/cea58426e0a8f4cfe330bc805f88d12ab2d3fc4e/testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentContainers.java#L18-L19

Reply:
Still presto-master

Reply:
Right, this is going to be updated with: https://github.com/trinodb/docker-images/pull/89.

I merged this too quickly. ; /

Reply:
https://github.com/trinodb/trino/pull/7008